### PR TITLE
[ENH] Merge the add-description changes

### PR DESCRIPTION
**Pull request by surchs**
Closes #

Changes proposed in this pull request:

-
-

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [ ] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Tests pass
- [x] Checks pass

For new features:
- [x] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: run_bagel_cli.sh [hunk 1]
```diff
@@ -23,10 +24,16 @@ fi
 echo $ds "$ds_name"
 if [ ! -e ${out} ]; then
 
+    echo Checking data dictionary for descriptions!
+    source ./venv/bin/activate
+    python3 add_description.py ${workdir}/participants.json
```
Comment (alyssadai) [inline run_bagel_cli.sh:29]: Do you think we could get rid of this section @surchs? I feel like `add_description.py` may more sense to run as a one-of before we run the current script - hardcoding the environment path is also not ideal.

I'll be updating the README for this repo as part of a PR to update the below `docker run` commands to align with our latest CLI changes. We could move these commands to the README instead (or, you could do that as part of this PR) to separate it from the functionality of the bagel CLI runner.

**Review by alyssadai — APPROVED** [review]
Thanks for the changes @surchs!

Just two questions from me, otherwise 🧑‍🍳

File: run_bagel_cli.sh (hunk 1 above)
Comment (surchs) [reply run_bagel_cli.sh:29]: I can't remember why I did it this way, but the one thing I am thinking of now is that you would have to remember resetting each datasets git repo after all is done if you want to run add_description.py before running run_bagel_cli.sh. 

feel free to change it, but I would stick to the workflow where we don't make any lasting local changes to the local clones of the OpenNeuroDataset-jsonld forks we're handling

File: silly.json
[the remark this answers is not in the document]
Comment (surchs) [reply silly.json:1]: I have a feeling that no, we don't need it

**Pull request merged**